framework/docker/evstack/reth/hyperlane_provider.go

```diff
@@ -97,3 +81,20 @@ func (n *Node) GetHyperlaneRelayerChainConfig(ctx context.Context) (hyperlane.Re
 
 	return cfg, nil
 }
+
+func (n *Node) GetHyperlaneWarpConfigEntry(ctx context.Context) (*hyperlane.WarpConfigEntry, error) {
+	cfg, err := n.GetHyperlaneRelayerChainConfig(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	return &hyperlane.WarpConfigEntry{
+		Type:                     "synthetic",
+		Owner:                    "0xaF9053bB6c4346381C77C2FeD279B17ABAfCDf4d",
+		Mailbox:                  cfg.Mailbox,
+		InterchainSecurityModule: cfg.InterchainSecurityModule,
+		Name:                     "wTIA",
+		Symbol:                   "TIA",
+		Decimals:                 6,
```

Comment:
This might need to be made more flexible in future, but we can see. 

Is this supposed to be doing the job of this command?
```
hyperlane warp init
```